services/generic/meter.md

```diff
@@ -1,91 +1,170 @@
+| `meter_cooling` | kWh                                           | Cooling meter.     |
+
+## Interfaces
+
+| Type | Interface                   | Value type | Properties                                            | Storage          | Description                                                                                                                                                                            |
+|------|-----------------------------|------------|-------------------------------------------------------|------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.meter.get_report        | string     |                                                       |                  | Value is an import/consumption `unit` defined in <br/>[`sup_units`](#service-properties) property. May not be supported by all meters. Empty value should request all supported units. |
+| in   | cmd.meter.get_export_report | string     |                                                       |                  | Value is a export/production `unit` defined in [`sup_export_units`](#service-properties) property.                                                                                     |
+| in   | cmd.meter.reset             | null       |                                                       |                  | Resets all historical readings.                                                                                                                                                        |
+| out  | evt.meter.report            | float      | `unit`, `prv_data`, `delta_t`, `direction`, `virtual` | `unit`           | Returns an import/consumption meter report for `unit` specified in properties.                                                                                                         |
+| out  | evt.meter.export_report     | float      | `unit`, `prv_data`, `delta_t`, `direction`            | `unit:direction` | Returns a export/production meter report for `unit` specified in properties.                                                                                                           |
+| -    |                             |            |                                                       |                  |                                                                                                                                                                                        |
+| in   | cmd.meter_ext.get_report    | null       |                                                       |                  | Requests an extended electricity report.                                                                                                                                               |
```

Comment:
Maybe here again unified format like format -> Request issue of "cmd.meter_ext.report"

Reply:
Answered in several replies.